lib/loader.js

```diff
@@ -3,7 +3,7 @@
 const fetch = require('node-fetch');
 const fs = require('fs');
 const path = require('path');
-const linter = require('./linter.js');
+const linter = require('./rules.js');
```

Comment:
Amazing work on this PR! I think this one line is the only concern: can we rename this from `const linter` to `const rules` ?

Reply:
argh, my bad. Pushed a new commit